### PR TITLE
Chapt 02 _ A

### PR DESCRIPTION
**Pull request by phrost101**

### CODE REVIEW

**Comment by phrost101** [comment]
Submissions

File: students/phrost/Lesson02/Fizzy_buz.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+while a <= 100:
+    if a % 3 == 0 and a % 5 != 0:
+        print("Fizz")
+    elif a % 3 != 0 and a % 5 == 0:
+        print("Buzz")
+    elif a % 3 == 0 and a % 5 == 0:
+        print("Fizz_Buzz");
+    else:
+        print(a)
+    a += 1
```
Comment (charlesrobison) [inline students/phrost/Lesson02/Fizzy_buz.py:12]: This works :)

File: students/phrost/Lesson02/print_grid_1.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+horz = ('+'+ 4 * '-'+ '+' + 4 * '-'+'+')
+vert = ('|' +  4*' ' + '|' + 4*' ' + '|')
+print(horz)
+print(vert,vert,vert,vert,sep='\n')
+print(horz)
+print(vert,vert,vert,vert,sep='\n')
+print(horz)
```
Comment (charlesrobison) [inline students/phrost/Lesson02/print_grid_1.py:7]: This works as well

File: students/phrost/Lesson02/print_grid_2.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    print(horz)
+    b=0
+    while b < a:
+        print(vert)
+        b += 1
+    print(horz)
+    
+print_grid(0)
+print_grid(1)
+print_grid(2)
```
Comment (charlesrobison) [inline students/phrost/Lesson02/print_grid_2.py:18]: Nice

File: students/phrost/Lesson02/print_grid_3.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+def print_grid_3(m,n):
```
Comment (charlesrobison) [inline students/phrost/Lesson02/print_grid_3.py:1]: Need a space after the comma here

File: students/phrost/Lesson02/print_grid_3.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    while a < m:     
+        b = 0
+        print(x)
+        while b < n:
+            print(y)
+            b += 1
+        a += 1
+    print(x)
+
+print_grid_3(2,3)
```
Comment (charlesrobison) [inline students/phrost/Lesson02/print_grid_3.py:14]: Need spaces after commas here too

**Pull request merged**